### PR TITLE
Signature Bracelet Component

### PR DESCRIPTION
**Pull request by nikitin2009**
Task https://leximaio.teamwork.com/#/tasks/28971631

### CODE REVIEW

File: assets/section-signature-bracelet.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  .signature-bracelet__gird-icon {
+    max-width: 280px;
+    margin: 0 auto 1.8rem;
+    display: flex;
+    align-items: center;
+  }
+  .signature-bracelet__gird-icon.hidden-mobile {
+    display: none;
+  }
+  .signature-bracelet__gird-icon-image {
```
Comment (olafghanizadeh) [inline assets/section-signature-bracelet.css:52]: Typo in classname 'gird'

File: assets/section-signature-bracelet.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+@media screen and (max-width: 749px) {
+  .signature-bracelet__gird-icon {
+    max-width: 280px;
+    margin: 0 auto 1.8rem;
+    display: flex;
+    align-items: center;
+  }
+  .signature-bracelet__gird-icon.hidden-mobile {
```
Comment (olafghanizadeh) [inline assets/section-signature-bracelet.css:49]: Typo

File: assets/section-signature-bracelet.css [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    margin-bottom: 6.2rem;
+  }
+  .signature-bracelet__heading {
+    margin-top: 0;
+    margin-bottom: 2.3rem;
+  }
+  .signature-bracelet__gird-icon {
+    text-align: center;
+  }
+  .signature-bracelet__gird-icon-image {
```
Comment (olafghanizadeh) [inline assets/section-signature-bracelet.css:74]: In mock it is centered on mobile. Add justify-self: center.

**Review by olafghanizadeh — CHANGES_REQUESTED** [review]

File: assets/section-signature-bracelet.css (hunk 3 above)
Comment (nikitin2009) [reply assets/section-signature-bracelet.css:74]: Not sure what you mean, and why add this. Is it not centered for you on mobile?

Comment (nikitin2009) [reply assets/section-signature-bracelet.css:74]: There are style for mobile only added for this element

**Pull request merged**